controllers/machinedeletionremediation_controller.go

```diff
@@ -225,7 +245,8 @@ func (r *MachineDeletionRemediationReconciler) Reconcile(ctx context.Context, re
 		return ctrl.Result{}, errors.Wrapf(err, "failed to save Machine's name and namespace")
 	}
 
-	log.Info("request machine deletion", "machine", machine.GetName())
+	commonevents.NormalEvent(r.Recorder, mdr, machineDeletionRequestedEventReason, machineDeletionRequestedEventMessage)
```

Comment:
this this event says "requested" (past tense) it should emitted after successful machine deletion